frontend/src/style/style.scss

```diff
@@ -1,2 +1,4 @@
 @import "tours";
+@import "landing";
+@import "imageUploader";
 @import '../../node_modules/rc-slider/assets/index.css';
```

Comment:
Put the slider on top to not let it override our styles. (That was also my mistake) 😈

Reply:
Done!